tests/integration/test_vectorizers.py

```diff
@@ -263,7 +277,10 @@ async def test_vectorizer_aembed(avectorizer):
 @pytest.mark.asyncio
 async def test_vectorizer_aembed_many(avectorizer):
     texts = ["This is the first test sentence.", "This is the second test sentence."]
-    embeddings = await avectorizer.aembed_many(texts)
+    if isinstance(avectorizer, VoyageAITextVectorizer):
+        embeddings = await avectorizer.aembed_many(texts)
+    else:
+        embeddings = await avectorizer.aembed_many(texts)
```

Comment:
why is this if-else block needed?

Reply:
Not sure, will remove

Reply:
@justin-cechmanek Yeah, seems like a mistake. Sorry about that. (i think originally i was planning to add further parameters, but in its state this condition is not needed)